packages/scandipwa/src/component/ProductCard/ProductCard.style.scss

```diff
@@ -146,16 +146,11 @@
     }
 
     &-Name {
-        line-height: var(--product-card-name-line-height);
```

Comment:
If you remove it, then please check if this variable is used anywhere. I guess we don't need to store redundant variables.